src/pages/Defi/components/EligibleSlider.tsx

```diff
@@ -17,16 +18,24 @@ export const EligibleSlider: React.FC<EligibleSliderProps> = ({ slidesToShow = 4
   const eligibleOpportunities = useAppSelector(selectAggregatedEarnUserStakingEligibleOpportunities)
   const renderEligibleCards = useMemo(() => {
     // opportunities with 1% APY or more
-    const filteredEligibleOpportunities = eligibleOpportunities
+    let filteredEligibleOpportunities = eligibleOpportunities
       .filter(o => bnOrZero(o.tvl).gt(50000) && bnOrZero(o.apy).gte(0.01))
-      .sort((a, b) => bn(b.apy).toNumber() - bn(a.apy).toNumber())
+      .sort((a, b) => bn(b.apy ?? '0').toNumber() - bn(a.apy ?? '0').toNumber())
       .slice(0, 5)
 
     const foxFarmingV9 = eligibleOpportunities.find(
       eligibleOpportunity =>
         eligibleOpportunity.contractAddress === ETH_FOX_STAKING_CONTRACT_ADDRESS_V9,
     )
 
+    const rfoxOpportunity = eligibleOpportunities.find(
+      eligibleOpportunity => eligibleOpportunity.provider === DefiProvider.rFOX,
+    )
+
+    if (rfoxOpportunity) {
+      filteredEligibleOpportunities = [rfoxOpportunity, ...filteredEligibleOpportunities]
+    }
```

Comment:
suggestion: revert as well as l.21 and move logic to the .filter() call

Reply:
Actually no, because the rFOX opportunity would always be the latest one as the APY is undefined, I wanted to put it first in the list to make it more visible